docker/s6/gogs/run

```diff
@@ -12,17 +11,16 @@ if ! test -d ~git/.ssh; then
 fi
 
 if ! test -f ~git/.ssh/environment; then
-    echo "GOGS_CUSTOM=/data/gogs" > ~git/.ssh/environment
-    chown git:git ~git/.ssh/environment
-    chown 600 ~git/.ssh/environment
+    echo "GOGS_CUSTOM=${GOGS_CUSTOM}" > ~git/.ssh/environment
+    chmod 600 ~git/.ssh/environment
 fi
 
-ln -sf /data/gogs/log  /app/gogs/log
-ln -sf /data/gogs/data /app/gogs/data
-ln -sf /data/gogs/conf /app/gogs/conf
+cd /app/gogs
+
+ln -sf /data/gogs/log  ./log
+ln -sf /data/gogs/data ./data
 
 chown -R git:git /data /app/gogs ~git/
 
 export USER
-export USERNAME
```

Comment:
Thanks, was weird that it did not happened when USERNAME was not set :/ Will test it